tests/drivers/uart/uart_elementary/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include "nrf54h20dk_nrf54h20_common.dtsi"
+
+
+&dut {
+	memory-regions = <&cpuapp_dma_region>;
+};
+
+&grtc {
```

Comment:
can be removed. Looks like copy-paste from uart_mix_fifo_poll test where it was used.